backend/database/migrations/2023_02_08_173512_add_taken_to_categories.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('categories', function (Blueprint $table) {
+            $table->boolean('taken')->default(0);
```

Comment:
What are you adding taken to categories? Meaning if user_1 took this category_1 and completed. Then user_2 cannot take this category_1?

Can you check your ERD if this was planned?

Reply:
I already deleted that on my integration and created a new migration for that sir :bow: